### ISSUE TITLE
Les livres de classes/études ne prennent plus le nom de la personne qui les craft

### ISSUE THREAD

**Issue by MirageNebulix** [Eowymos/TerraUo#382]
Les livres de classes/études ne prennent plus le nom de la personne qui les craft, alors que sur des anciens on voyait le nom du fabriquant.

**Comment by animatrice-hanson** [comment]
Est-ce que tu peux me dire exactement ceux qui ne fonctionnait pas? Car je viens d'en tester quelques uns et tout semble bien fonctionner. As-tu bien l'option Marquage: Marquer?

**Comment by MirageNebulix** [comment]
Oui, je l'ai même enlevée et remise au cas ou il fallait un 'reset'. Ce sont les livres dans le menu de bricolage- Assemblage/28.Livre d'étude/classe vierge (1). Tous ceux que j'ai crafté récemment aucun n'a pris le nom de mon perso et c'est bien indiqué Marquage: Marquer.

**Comment by animatrice-hanson** [comment]
Ah ! Les livres vierges oui ! C'est un livre vierge donc il n'a pas "d'auteur", dès que tu utilises le scribe's pen pour en faire un livre, ton nom sera marqué et tu pourras ajouter ensuite une description. La plupart des objets dans le menu du bricoleur sont des composantes pour créer un autre objet et donc ne sont pas marquables ou descriptibles. Ceci n'est donc pas considéré comme un bug mais plutôt une suggestion de pouvoir marquer et décrire les outils et petits objets du menu bricoleur. Je laisse ouvert comme suggestion pour l'équipe technique.

**Comment by MirageNebulix** [comment]
Non, même après avoir utilisé le scribe's pen dessus le nom de mon perso n'est toujours pas indiqué.

**Comment by MirageNebulix** [comment]
Je viens de crafter deux statues, et bien que l'option marquage soit a Marquer (j'ai testé en l'enlevant et le remettant) le nom de ma pj ne s'affiche pas sur la statue.

**Comment by MirageNebulix** [comment]
Grimoires : Ils prennent bien le nom du crafter, mais impossible de leur mettre une description.
